spec/example_app/app/controllers/admin/stats_controller.rb

```diff
@@ -0,0 +1,10 @@
+module Admin
+  class StatsController < Admin::ApplicationController
+    def index
+      @stats = {
```

Comment:
How would an end user know to provide something in this hash called `@stats`?

Reply:
What do you mean? This was a simple example for the demo. It could be anything, really.